### PR TITLE
Refresh file metadata save

### PR DESCRIPTION
**Pull request by AdarshdeepCheema**
## Proposed changes

This PR addresses Issue: [*Link to Github issue within https://github.com/zowe/zlux/issues* if any]
https://github.com/zowe/zlux/issues/660

This PR depends upon the following PRs:
https://github.com/zowe/zlux-file-explorer/pull/124

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Change in a documentation
- [ ] Refactor the code 
- [ ] Chore, repository cleanup, updates the dependencies.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## PR Checklist
Please delete options that are not relevant.
- [ ] If the changes in this PR are meant for the next release / mainline, this PR targets the "staging" branch.
- [x] My code follows the style guidelines of this project (see: [Contributing guideline](https://github.com/zowe/zlux/blob/master/CONTRIBUTING.md))
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes
- [ ] video or image is included if visual changes are made
- [x] Relevant update to CHANGELOG.md
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works, or describe a test method below

### CODE REVIEW

**Comment by DivergentEuropeans** [comment]
This needs a CHANGELOG so please update this PR with that, along with the other two changes that were already merged if they weren't included

**Comment by DivergentEuropeans** [comment]
https://github.com/zowe/zlux-file-explorer/pull/124#pullrequestreview-716472672

**Review by DivergentEuropeans — APPROVED** [review]

File: webClient/src/app/editor/project-tree/project-tree.component.ts [hunk 1]
```diff
@@ -184,13 +184,22 @@ export class ProjectTreeComponent {
...
+    });
+
     this.editorControl.createDirectory.subscribe(pathAndName => {
       this.fileExplorer.createDirectory(pathAndName);
     });
 
     this.editorControl.toggleFileTreeSearch.subscribe(() => {
       this.fileExplorer.toggleSearch();
     })
+
```
Comment (DivergentEuropeans) [inline webClient/src/app/editor/project-tree/project-tree.component.ts:198]: You added a duplicate here by accident (maybe from Git cleanup things)

File: webClient/src/app/shared/editor-control/editor-control.service.ts [hunk 2]
```diff
@@ -53,7 +53,9 @@ export class EditorControlService implements ZLUX.IEditor, ZLUX.IEditorMultiBuff
   public closeAllFiles: EventEmitter<string> = new EventEmitter();
   public undoCloseAllFiles: EventEmitter<string> = new EventEmitter();
   public activeDirectory = '';
+  public refreshFileMetadatdaByPath: EventEmitter<string> = new EventEmitter();
   public deleteFile: EventEmitter<string> = new EventEmitter();
+  public refreshFileMetadatdaByPath: EventEmitter<string> = new EventEmitter();
```
Comment (DivergentEuropeans) [inline webClient/src/app/shared/editor-control/editor-control.service.ts:58]: You added a duplicate here by accident (maybe from Git cleanup things)

**Review by DivergentEuropeans — CHANGES_REQUESTED** [review]

File: webClient/src/app/shared/editor-control/editor-control.service.ts (hunk 2 above)
Comment (AdarshdeepCheema) [reply webClient/src/app/shared/editor-control/editor-control.service.ts]: Fixed

**Comment by AdarshdeepCheema** [comment]
Will be done on https://github.com/zowe/zlux-editor/pull/222

**Pull request closed without merging**